app/src/features/FhirAttributePanel/FhirAttributePanel.tsx

```diff
@@ -50,9 +61,13 @@ const FhirAttributePanel = (): JSX.Element => {
     }
   };
 
+  if (isFetching) {
```

Comment:
why not putting isUninitialized here ?

Reply:
I just thought that CircularProgress should only be shown when fetching data

isUninitialized can be true when data is fetched, especially when we change the mapping and that the page is still mounted